src/SixLabors.Fonts/FontDescription.cs

```diff
@@ -27,8 +28,9 @@ internal FontDescription(string fontName, string fontFamily, string fontSubFamil
         /// <param name="nameTable">The name table.</param>
         /// <param name="os2">The os2.</param>
         /// <param name="head">The head.</param>
-        internal FontDescription(NameTable nameTable, OS2Table? os2, HeadTable? head)
-            : this(nameTable.FontName, nameTable.FontFamilyName, nameTable.FontSubFamilyName, ConvertStyle(os2, head))
+        /// <param name="culture">The culture to load metadata in.</param>
+        internal FontDescription(NameTable nameTable, OS2Table? os2, HeadTable? head, CultureInfo culture)
+            : this(nameTable.FontName(culture), nameTable.FontFamilyName(culture), nameTable.FontSubFamilyName(culture), ConvertStyle(os2, head))
```

Comment:
I think we could also have a `NameTable` property with `internal` access level, so `FontCollection` can access various valus from `NameTable`, through `FontDescription`.